arch/renesas/Kconfig

```diff
@@ -61,21 +61,26 @@ config ARCH_RX65N
 	bool
 	default n
 	select CYGWIN_WINTOOL if WINDOWS_CYGWIN
+	select ARCH_HAVE_SETJMP
 
 config ARCH_RX65N_RSK1MB
```

Comment:
should we only use ARCH_RX65N to describe the sub arch, and ARCH_RX65N_RSK1MB, ARCH_RX65N_RSK2MB and ARCH_RX65N_GRROSE as the chip variant? so we can check ARCH_RX65N in various place.

Reply:
Yeah, I was thinking something like that would be cleaner. I will try and do that.